src/templates/default.json

```diff
@@ -406,141 +399,98 @@
           "leftIcon": "",
           "size": "md",
           "key": "submit",
-          "tableView": false,
           "label": "Submit",
-          "input": true
+          "input": true,
+          "type": "button"
         }
       ]
     }
   },
   "actions": {
-    "user:role": {
-      "title": "Role Assignment",
-      "name": "role",
-      "priority": 1,
-      "handler": ["after"],
-      "method": ["create"],
-      "form": "user",
-      "settings": {
-        "association": "new",
-        "type": "add",
-        "role": "authenticated"
-      }
+    "guardianSave": {
+      "title": "Save Submission",
+      "name": "save",
+      "form": "guardian",
+      "handler": ["before"],
+      "method": ["create", "update"],
+      "priority": 11,
+      "settings": {}
     },
-    "user:save": {
+    "adminSave": {
       "title": "Save Submission",
       "name": "save",
-      "form": "user",
+      "form": "admin",
       "handler": ["before"],
       "method": ["create", "update"],
-      "priority": 10
+      "priority": 11,
+      "settings": {}
     },
-    "userLogin:login": {
+    "userLogin": {
       "name": "login",
       "title": "Login",
       "form": "userLogin",
       "priority": 2,
       "method": ["create"],
       "handler": ["before"],
       "settings": {
-        "resources": ["user"],
+        "resources": ["admin"],
         "username": "email",
-        "password": "password",
-        "allowedAttempts": 5,
-        "attemptWindow": 30,
-        "lockWait": 1800
+        "password": "password"
       }
     },
-    "userRegister:save": {
+    "guardianRegisterSave": {
       "title": "Save Submission",
       "name": "save",
-      "form": "userRegister",
+      "form": "guardianRegister",
       "handler": ["before"],
-      "method": ["create", "update"],
-      "priority": 11,
+      "method": ["create"],
+      "priority": 10,
       "settings": {
-        "resource": "user",
+        "resource": "guardian",
         "fields": {
           "email": "email",
           "password": "password"
         }
       }
     },
-    "userRegister:login": {
+    "guardianRegisterLogin": {
```

Comment:
what does `guardianRegisterLogin` mean? Register and Login seem like two different things?